tests/test_core.py

```diff
@@ -9143,8 +9145,13 @@ def setUp(self):
 core3 = make_run('core3', emcc_args=['-O3'])
 cores = make_run('cores', emcc_args=['-Os'])
 corez = make_run('corez', emcc_args=['-Oz'])
-core64 = make_run('core64', emcc_args=['-O0', '-g3'],
-                  settings={'MEMORY64': 2}, env=None, node_args='--experimental-wasm-bigint')
+
+# MEMORY64=1
+wasm64 = make_run('wasm64', emcc_args=['-O0', '-g3'],
+                  settings={'MEMORY64': 1}, require_v8=True, v8_args=['--experimental-wasm-memory64'])
+# MEMORY64=2, or "lowered"
+wasm64l = make_run('wasme64', emcc_args=['-O0', '-g3'],
```

Comment:
why is this `wasme64` and not `wasm64l`?

Reply:
oops typo